org.audiveris.audiveris.yml

```diff
@@ -0,0 +1,982 @@
+
+app-id: org.audiveris.audiveris
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+build-options:
+  append-path: "/usr/lib/sdk/openjdk17/bin"
+command: Audiveris
+finish-args:
+  # Network access
+  - --share=network
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Wayland access
+  - --socket=wayland
```

Comment:
does it really support wayland? If yes then line 16 should be fallback-x11. otherwise line 18 should be removed.

Reply:
OK, I just copied this part from somewhere else I IIRC. 
I'll remove wayland for now.